src/server/rest/v1/service/AuthService.ts

```diff
@@ -119,7 +120,7 @@ export default class AuthService {
 
   public static async handleRegisterUser(action: ServerAction, req: Request, res: Response, next: NextFunction): Promise<void> {
     // Filter
-    const filteredRequest = AuthValidator.getInstance().validateAuthSignOnReq(req.body);
+    const filteredRequest = AuthValidatorStorage.getInstance().validateAuthSignOn(req.body);
```

Comment:
Why do you use AuthValidatorStorage, it's used for storing data in DB, use AuthValidator instead!